### ISSUE TITLE
board.setBoardProperty not passing along value.

### ISSUE THREAD

**Issue by lukaswelinder** [floralvikings/jira-connector#179]
It looks like the `setBoardProperty` function does not pass along a value for `body` on the [request options](https://github.com/floralvikings/jira-connector/blob/master/api/board.js#L477).

Jira returns a `400` error with the message `"The property value can not be empty."`.

Possible fix:

```js
  this.setBoardProperty = function (opts, callback) {
    var options = {
      uri: this.jiraClient.buildAgileURL('/board/' + opts.boardId + '/properties/' + opts.propertyKey),
      method: 'PUT',
      json: true,
      followAllRedirects: true,
      body: opts.value
    };

    return this.jiraClient.makeRequest(options, callback);
  }
```

Such that the options would look like this:

```js
{
  boardId: 1,
  propertyKey: '<string>',
  value: '<string>'
}
```

But this is assuming that the API behaves similar to the other properties API (project/issue/etc). For some reason, the [documentation](https://developer.atlassian.com/cloud/jira/software/rest/#api-rest-agile-1-0-board-boardId-properties-propertyKey-put) for the endpoint does not mention a payload.

**Comment by MrRefactoring** [comment]
It is rather strange that in the documentation there is no mention of the body parameter 🤔. And you can look for an issue about this problem? For example, [here](https://jira.atlassian.com/projects/JRASERVER/issues) or elsewhere. I want to understand this problem in more detail. Maybe this is also found in other endpoints?

**Comment by MrRefactoring** [comment]
Fixed in version 2.16.0. Thanks for your contribution!
